docs/Behaviour - Server Side.md

```diff
@@ -31,23 +31,27 @@ If an active Receiver becomes unable to handle Flow, it is REQUIRED to take the
 
 ### Properties
 
-The `/properties` endpoint shows properties of the Input. In particular, it MAY contain information parsed from Base EDID and Effective EDID if they are present and the Device is capable to parse EDID binaries.
+The `/properties` endpoint shows properties of the Input. In particular, it MAY contain information parsed from Base EDID and Effective EDID if the Input supports EDID and the Node is capable to parse EDID binaries.
 
 ### Base EDID
 
-`GET /edid/base` and `PUT /edid/base` operations allow a client to download and replace the Base EDID respectively if it exists. If the Base EDID for the Input changes and it is associated with any Senders, then all of the Senders in question MUST update their versions (in registered mode this MUST update the registered resources).
+The initial state of Base EDID is such an EDID which this Input presents to the upstream counterpart if Active Constraints are empty.
+
+`GET /edid/base` and `PUT /edid/base` operations allow a client to download and replace the Base EDID respectively if it exists. `DELETE /edid/base` resets Base EDID to the initial state. If the Base EDID for the Input changes and it is associated with any Senders, then all of the Senders in question MUST update their versions (in registered mode this MUST update the registered resources).
```

Comment:
There will be no way of knowing of a PUT has been performed on /edid/base if we return the manufacturer default base EDID after a delete to /edid/base. Is it appropriate according to REST rules? Another approach to get the manufacturer default EDID would be a GET on /edid/effective with empty /constraints/active

Reply:
You're perfectly right. I replaced default Base EDID with No Content.

Reply:
2021/11/22 activity group call: marked as resolved.